pom.xml

```diff
@@ -56,4 +78,16 @@
             <email>janarioliver@gmail.com</email>
         </developer>
     </developers>
+
+    <build>
+        <pluginManagement>
```

Comment:
pluginManagment doesn't make any sense in non multi-module project. why do you need it?